### PR TITLE
Added fractional knapsack 

### PR DESCRIPTION
**Pull request by ritikverma2000**
I have added fractional knapsack code under greedy. Given the weights and values of n items, we need to put these items in a knapsack of capacity W to get the maximum total value in the knapsack. In Fractional Knapsack, we can break items for maximizing the total value of the knapsack.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -51,7 +51,8 @@ If you want to uninstall algorithms, it is as simple as:
     $ pip3 uninstall -y algorithms
 
 ## List of Implementations
-
+- [Greedy](algorithms/arrays)
```
Comment (ericklarac) [inline README.md:54]: The elements are listed in alphabetical order. Could you add it below dp?

File: algorithms/Greedy/fractionalKnapsack.py [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+		return self.cost < other.cost
+
+# Greedy Approach
+
+
+class FractionalKnapSack:
+
+	"""Time Complexity O(n log n)"""
+	@staticmethod
+	def getMaxValue(wt, val, capacity):
```
Comment (ericklarac) [inline algorithms/Greedy/fractionalKnapsack.py:24]: Use snake case

**Review by ericklarac — CHANGES_REQUESTED** [review]
Also add some unit tests

File: README.md (hunk 1 above)
Comment (ritikverma2000) [reply README.md:54]: ok no issues I will do it

File: algorithms/Greedy/fractionalKnapsack.py (hunk 2 above)
Comment (ritikverma2000) [reply algorithms/Greedy/fractionalKnapsack.py:24]: ok

**Comment by ritikverma2000** [comment]
> Also add some unit tests

what do you mean by some unit test cases?

**Comment by ericklarac** [comment]
> > Also add some unit tests
> 
> what do you mean by some unit test cases?

To add test cases under the test folder, check [this PR](https://github.com/keon/algorithms/pull/711/files) as guide

**Comment by ritikverma2000** [comment]
> > > Also add some unit tests
> > 
> > 
> > what do you mean by some unit test cases?
> 
> To add test cases under the test folder, check [this PR](https://github.com/keon/algorithms/pull/711/files) as guide

I am new to open-source I don't know how to proceed with the test folder.

**Review by ericklarac — COMMENTED** [review]
> > > > Also add some unit tests
> > > 
> > > 
> > > what do you mean by some unit test cases?
> > 
> > 
> > To add test cases under the test folder, check [this PR](https://github.com/keon/algorithms/pull/711/files) as guide
> 
> I am new to open-source I don't know how to proceed with the test folder.

Refer to [python unittest](https://docs.python.org/3/library/unittest.html), this is to add some test cases for your code/functions.

File: algorithms/Greedy/fractionalKnapsack.py [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+		return self.cost < other.cost
+
+# Greedy Approach
+
+
+class FractionalKnapSack:
+
+	"""Time Complexity O(n log n)"""
+	@staticmethod
+	def get_Max_Value(wt, val, capacity):
```
[the remark this answers is not in the document]
Comment (ritikverma2000) [reply algorithms/Greedy/fractionalKnapsack.py:24]: ok

**Comment by ritikverma2000** [comment]
> > > > > Also add some unit tests
> > > > 
> > > > 
> > > > what do you mean by some unit test cases?
> > > 
> > > 
> > > To add test cases under the test folder, check [this PR](https://github.com/keon/algorithms/pull/711/files) as guide
> > 
> > 
> > I am new to open-source I don't know how to proceed with the test folder.
> 
> Refer to [python unittest](https://docs.python.org/3/library/unittest.html), this is to add some test cases for your code/functions.

I have made all necessary changes as asked by you pls allow me to merge my pull request.

**Comment by ritikverma2000** [comment]
> Also add some unit tests

**Comment by ritikverma2000** [comment]
Pls add hacktoberfest label also to this pull request

**Comment by ritikverma2000** [comment]
pls allow me to merge

**Comment by ritikverma2000** [comment]
thanks for merging

**Pull request closed without merging**